src/System.Threading.Tasks.Dataflow/src/Base/DataflowBlockOptions.cs

```diff
@@ -170,6 +177,13 @@ public string NameFormat
                 _nameFormat = value;
             }
         }
+
+        /// <summary>Gets whether ordered processing should be enforced on a block's handling of messages.</summary>
```

Comment:
Can you be a bit clearer in the docs:
What order?  When this is false what is the expected behavior?  What's the tradeoff for setting this one way or the other.
"Force" seems to indicate that the default is false when in fact it is true.  Perhaps "ProcessInOrder" might be a better name?